### PR TITLE
all: update for patch release 3.13.2

### PR DESCRIPTION
**Pull request by keegancsmith**
Updates docs and updatecheck.

Note: CI will fail since not all upstream changes are ready. Will resubmit CI once ready to merge.

Part of https://github.com/sourcegraph/sourcegraph/issues/8939
Part of https://github.com/sourcegraph/sourcegraph/issues/8940

### CODE REVIEW

File: doc/index.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ We actively maintain the two most recent monthly releases of Sourcegraph.
 Upgrades should happen across consecutive minor versions of Sourcegraph. For example, if you are
 running Sourcegraph 3.1 and want to upgrade to 3.3, you should upgrade to 3.2 and then 3.3.
 
-> The Docker server image tags follow SemVer semantics, so version 3.8 can be found at `sourcegraph/server:3.13.1`. You can see the full list of tags on our [Docker Hub page](https://hub.docker.com/r/sourcegraph/server/tags).
+> The Docker server image tags follow SemVer semantics, so version 3.8 can be found at `sourcegraph/server:3.13.2`. You can see the full list of tags on our [Docker Hub page](https://hub.docker.com/r/sourcegraph/server/tags).
```
Comment (unknwon) [inline doc/index.md]: This line shouldn't be changed, nevertheless we change it in every single release :D

Comment (keegancsmith) [reply doc/index.md]: we use a script to update :) Will see if we can make the script smarter/the text different. For now I'll just update so it makes sense.

Comment (keegancsmith) [reply doc/index.md]:
```suggestion
> The Docker server image tags follow SemVer semantics, so version 3.13.2 can be found at `sourcegraph/server:3.13.2`. You can see the full list of tags on our [Docker Hub page](https://hub.docker.com/r/sourcegraph/server/tags).
```

Comment (unknwon) [reply doc/index.md]: SGTM!

**Pull request merged**